src/components/editor/PropertyTemplate.jsx

```diff
@@ -35,5 +36,14 @@ class PropertyTemplate extends Component {
   }
 }
 
+PropertyTemplate.propTypes = {
+  propertyLabel: PropTypes.string,
+  propertyURI: PropTypes.string,
```

Comment:
I suspect some of these are required?  esp `propertyURI` and maybe `propertyLabel`?  and `type`?